liqd_lunch/frontend/src/RestaurantsList.js

```diff
@@ -22,12 +22,18 @@ class  RestaurantsList  extends  Component {
 
     componentDidMount() {
         var  self  =  this;
-        restaurantsService.getRestaurants().then(function (result) {
-            self.setState({
-              restaurants:  result.data,
-              nextPageURL:  result.nextlink,
-            })
-        });
+        if (this.timeForLunch()) {
+            alert('Voting is closed, it is LUNCH TIME!');
```

Comment:
Basically cool for now - but as we want to present things tomorrow, maybe we can go for:
```
if (confirm('Voting is closed, it is LUNCH TIME! Do you want to see the result?')) {
   window.location.href = "/resistances";
} else {
  (the block below)
  console.log('Thing was not saved to the database.');
}
```